src/lualib/Iterator.ts

```diff
@@ -27,6 +27,6 @@ function __TS__Iterator<T>(
         const iterator = iterable[Symbol.iterator]();
         return [__TS__IteratorIteratorStep, iterator];
     } else {
-        return ipairs(iterable as readonly T[]);
+        return ipairs(iterable as readonly T[]) as any;
```

Comment:
What is the reasoning behind this `as any?`

Reply:
`ipairs` returns a `LuaTupleIterator` which is incompatible with `__TS__Iterator` signature. Generally this should be resolved with 880